src/FileSystemUtilities.js

```diff
@@ -51,10 +51,7 @@ export default class FileSystemUtilities {
 
   @logger.logifyAsync()
   static rimraf(filePath, callback) {
-    ChildProcessUtilities.exec("npm bin", {cwd: __dirname}, (err, npmBinPath) => {
-      if (err) return callback(err);
-      ChildProcessUtilities.spawn(join(npmBinPath.trim(), "rimraf"), [filePath], {}, callback);
-    });
+    ChildProcessUtilities.spawn(require.resolve("rimraf/bin"), [filePath], {}, callback);
```

Comment:
To your idea, is `./.bin/rimraf` guaranteed to be set in the same node_modules folder where `rimraf` is installed? If so, then yeah I like your idea. Would we need to use `path.resolve` in that case?
```js
path.resolve(require.resolve("rimraf"), "../.bin/rimraf")
```